client/my-sites/site-settings/analytics/form-google-analytics.jsx

```diff
@@ -89,7 +90,7 @@ export const GoogleAnalyticsForm = ( props ) => {
 		recordSupportLinkClick,
 		setDisplayForm,
 	};
-	if ( props.siteIsJetpack ) {
+	if ( props.siteIsJetpack && ! props.siteIsFreeAtomic ) {
```

Comment:
I used `isFreeAtomic` rather than `isAtomic` here since the Jetpack version of this form is currently available to Atomic sites.